lib/js/components/Banner/Banner.vue

```diff
@@ -28,50 +25,38 @@
 								:color="BUTTON_COLORS.NEUTRAL"
 								:type="BUTTON_TYPES.OUTLINED"
 								:size="BUTTON_SIZES.SMALL"
-								@click.native="$emit('button-clicked')"
+								@click="$emit('button-clicked')"
 								>{{ buttonText }}
 							</ds-button>
 						</div>
-						<div
-							v-if="$slots.rightSlot && $slots.rightSlot.length > 0"
-							class="ds-banner__rightSlot"
-						>
+						<div v-if="$slots.rightSlot" class="ds-banner__rightSlot">
 							<slot name="rightSlot" />
 						</div>
 					</div>
 				</div>
 			</div>
-			<div
-				v-if="$slots.expandedText && $slots.expandedText.length > 0"
```

Comment:
vue3 return slots as function, so we can't check `length`. Fixed `v-if` on template in stories